lib/web_ui/lib/src/engine/platform_views/message_handler.dart

```diff
@@ -105,28 +103,28 @@ class PlatformViewMessageHandler {
   /// Handle a `dispose` Platform View message.
   ///
   /// This will clear the cached information that the framework has about a given
-  /// `viewId`, through the [_contentManager].
+  /// `platformViewId`, through the [_contentManager].
   ///
   /// Once that's done, the dispose call is delegated to the [_disposeHandler]
   /// function, so the active rendering backend can dispose of whatever resources
   /// it needed to get ahold of.
   ///
   /// This function should always `callback` with an empty success envelope.
   void _disposePlatformView(
-    int viewId,
-    _PlatformMessageResponseCallback callback,
-  ) {
+    _PlatformMessageResponseCallback callback, {
+    required int platformViewId,
+  }) {
     // The contentManager removes the slot and the contents from its internal
     // cache, and the DOM.
-    _contentManager.clearPlatformView(viewId);
+    _contentManager.clearPlatformView(platformViewId);
 
     callback(_codec.encodeSuccessEnvelope(null));
   }
 
-  /// Handles legacy PlatformViewCalls that don't contain a `flutterViewId`.
+  /// Handles legacy PlatformViewCalls that don't contain a Flutter View ID.
   ///
   /// This is transitional code to support the old platform view channel. As
-  /// soon as the framework code is updated to send the `flutterViewId`, this
+  /// soon as the framework code is updated to send the Flutter View ID, this
   /// method can be removed.
   void handleLegacyPlatformViewCall(
```

Comment:
I totally forgot about this method. Good repurposing though!

Reply:
You must've been looking at an individual commit from the PR 🙂 This method is new.